.github/workflows/nolints.yml

```diff
@@ -54,14 +59,15 @@ jobs:
             bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
 
       - name: update nolints.json and style-exceptions.txt
+        continue-on-error: true
```

Comment:
Actually, I'm not sure this is the best solution, as even if this allows the workflow to proceed, the job will get a red x and notify whoever modified the workflow last.  We could add `continue-on-error` to the job (as we do in the `bot_fix_style_*` workflows), but that will cover up errors in other steps of the workflow.

@grunweg Would it make sense to have `lint-style` not return an nonzero exit code when linting errors are detected when run in `--update` or `--fix` modes?

Reply:
I think it makes sense, yes: update or fix mode are *supposed* to fix errors (so returning zero if something changed seems morally fine); the lint-style check runs without these arguments, so is not affected